salt/orch/update.sls

```diff
@@ -434,10 +433,10 @@ remove-caasp-fqdn-grain:
     - require:
       - admin-wait-for-services
 
-masters-remove-update-grain:
+remove-update-grain:
```

Comment:
is this similar to what we do in the `{{ worker_id }}-remove-update-grain` sls? now that it's not tied to `is_master_tgt` anymore